lib/solargraph/api_map.rb

```diff
@@ -103,19 +105,7 @@ def catalog bundle
       reqs.merge bundle.workspace.config.required
       local_path_hash.clear
       unless bundle.workspace.require_paths.empty?
-        file_keys = new_map_hash.keys
-        workspace_path = Pathname.new(bundle.workspace.directory)
-        reqs.delete_if do |r|
-          bundle.workspace.require_paths.any? do |base|
-            pn = workspace_path.join(base, "#{r}.rb").to_s
-            if file_keys.include? pn
-              local_path_hash[r] = pn
-              true
-            else
-              false
-            end
-          end
-        end
+        local_path_hash = @require_diff.unresolved_requires(bundle, reqs.clone, new_map_hash)
```

Comment:
```suggestion
        local_path_hash.merge! @require_diff.unresolved_requires(bundle, reqs.clone, new_map_hash)
```

Reply:
Good catch this should be fixed now as well as a few other potentially problems.